about2.html

```diff
@@ -0,0 +1,133 @@
+        </ul>
+    </div>
+    
+    <main>
+        <section class="cover-about">
+            <div class="container program_titles">
+                <h1>CREATING HIGH-PERFORMANCE
+                    TEAMS CONFERENCE
+                    BOGOTÁ 2022</h1>
+
+                <hr>
+                <div class="cover_about">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc dapibus diam sed pulvinar congue.
+                    Vivamus
+                    sit amet nisi a tellus interdum placerat vitae sit amet dolor. Praesent placerat hendrerit mi, in
+                    ultrices lectus vulputate et. Aenean lacinia feugiat ultrices. Nullam dapibus tortor at purus
+                    eleifend
+                    bibendum. Etiam tristique, quam eleifend auctor bibendum, nunc massa condimentum odio, et facilisis
+                    elit
+                    odio in felis. Vivamus tincidunt est tempus orci tempus, non cursus ex maximus. Pellentesque
+                    habitant
+                    morbi tristique senectus et netus et malesuada fames ac turpis egestas.
+
+                    Fusce at dui a mauris sollicitudin accumsan. Aenean gravida eget metus ut porta. Ut at varius eros.
+                    Aenean ac lectus molestie, varius ante at, faucibus dolor. Sed convallis lectus turpis, ac egestas
+                    libero accumsan ac. Vivamus vulputate massa nec sapien cursus facilisis. Quisque turpis diam,
+                    porttitor
+                    sed metus a, sodales placerat sem. Duis nec ipsum et ipsum lobortis lobortis. Fusce euismod blandit
+                    neque non accumsan. Sed vulputate venenatis nibh, et sagittis sem pretium id. Nulla at dapibus
+                    velit.
+                    Proin nec nisi porttitor, egestas sapien a, elementum.
+                </div>
+
+                <p>2022.02.18(SAT)- 20(MON) </p>
+                <p>Gran Salon Corferias, Bogotá-Colombia</p>
+            </div>
+
+        </section>
+        <section class="news">
+            <div class="container program_titles" >
+                <h4><strong>Creating High-Performance Teams Conference Bogotá Logo</strong></h4>
+                <hr>
+
+                <p>The Bogota brand, approved by the Council, has identified the capital over the last decade and was
+                    created in 2008 with the participation of the Mayor's Office of Bogota, through the District
+                    Institute
+                    of
+                    Tourism, the IDT, the Bogota Chamber of Commerce, the CCB, and Invest Bogota, with the vision of
+                    creating a
+                    city strategy that would position the city as a tourist destination, investment and
+                    business articulation at national and international level.
+                </p>
+                <img src="./images/logo_bogota.jpg" alt="logo" />
```

Comment:
- [ ] As the previous reviewer suggested, the pages should look identical to the original design. Please include adequate spacing around this section so it looks identical to the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169634864-b0a6e1b8-eb12-4e88-bf1f-6ae8157f7f6b.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/169634897-fa3ac6c0-00ef-4081-8afd-9af58c131432.png)